### PR TITLE
Added docker-compose.yaml in the root folder

### PR DESCRIPTION
**Pull request by willianpc**
This PR introduces a `docker-compose.yaml` file in the root of the project, initially exposing only PostreSQL.
The idea is to be able to run integration tests locally, which currently are expected to happen only in Circle CI.

Once this PR is merged, the following steps can be followed to run pgx integration tests locally.

Run these commands from the root folder of the project:

```bash
$ docker-compose up -d postgres
$ make integration
$ docker-compose stop postgres
```

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+
+  postgre:
```
Comment (sanojsubran) [inline docker-compose.yaml:5]: Nitpicking: 'postgre' intentionally ?

**Review by sanojsubran — APPROVED** [review]
Looks good

File: docker-compose.yaml (hunk 1 above)
Comment (sanojsubran) [inline docker-compose.yaml:5]:
```suggestion
  postgres:
```

Comment (willianpc) [reply docker-compose.yaml:5]: Good point. What is the right name?
I checked the official page before writing that: https://www.postgresql.org/
What do you suggest to name it? Maybe although the real name is `postgre` people call it something else?

Comment (willianpc) [reply docker-compose.yaml:5]: Please, read my reply in the previous comment

Comment (sanojsubran) [reply docker-compose.yaml:5]: It is absolutely fine if you purposefully wrote it. Kindly ignore my comment.

Comment (willianpc) [reply docker-compose.yaml:5]: [Wikipedia](https://en.wikipedia.org/wiki/PostgreSQL) tells me that `Postgres` is also correct:

```
PostgreSQL also known as Postgres...
```

I will change it :)

Comment (sanojsubran) [reply docker-compose.yaml]: Got it ! Kindly ignore this.

**Pull request merged**